metricbeat/module/kubelet/container/_meta/data.json

```diff
@@ -0,0 +1,78 @@
+{
+    "@timestamp": "2017-04-06T15:29:27.150Z",
+    "beat": {
+        "hostname": "beathost",
+        "name": "beathost",
+        "version": "6.0.0-alpha1"
+    },
+    "kubelet": {
+        "container": {
+            "cpu": {
+                "usage": {
+                    "corenanoseconds": 3305756719,
```

Comment:
Is this nano seconds? If yes, we should use `.ns`: https://www.elastic.co/guide/en/beats/libbeat/5.1/event-conventions.html